argoverse/evaluation/eval_detection.py

```diff
@@ -0,0 +1,271 @@
+# <Copyright 2020, Argo AI, LLC. Released under the MIT license.>
+import argparse
+import logging
+import os
+from collections import defaultdict
+from dataclasses import dataclass
```

Comment:
hmm tested this on the server and got a no module error for this dataclasses import. Pip install dataclasses explicitly fixed that though.  Seems like dataclasses is build in for python 3.7+ but not before that? It does works fine for my local machine with python 3.7 Should we add that to the required package?

Reply:
Yeah, I figured that would be an issue. They backported `dataclasses` to Python 3.6 after the 3.7 release. Any chance we could deprecate Python 3.6 and support 3.7+? It is the next version that will be EOL (3.5 will be EOL by the end of the month).

Reply:
that's temping. I feel like some people will still using it for awhile though,

Reply:
We could import the backport ... but I would be curious to discuss how many issues this would realistically cause (if we deprecated 3.6)

Reply:
bump

Reply:
@johnlambert-argo what's the call on this?